### ISSUE TITLE
Not found bootstrap.php file

### ISSUE THREAD

**Issue by henryonsoftware** [brefphp/symfony-messenger#36]
In the document says `consumer.php` has this require 
```require dirname(__DIR__) . '/config/bootstrap.php';```

But I can't find `bootstrap.php` file in config folder.

**Comment by mnapoli** [comment]
Oh has this file disappeared from Symfony? Which version are you using? (maybe you are using an old one, in which case you may want to include `vendor/autoload.php` instead?)

**Comment by henryonsoftware** [comment]
I'm using `v5.1.7` and it doesn't have that file from original source.

**Comment by mnapoli** [comment]
Indeed, it seems that in 4.4 [the file exists](https://github.com/symfony/recipes/tree/master/symfony/framework-bundle/4.4/config) but in [5.1](https://github.com/symfony/recipes/tree/master/symfony/framework-bundle/5.1/config) it doesn't exist anymore.

If we take a look at [5.1's `index.php`](https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/5.1/public/index.php) I think that line can be replaced with:

```php
require dirname(__DIR__).'/vendor/autoload.php';

(new Dotenv())->bootEnv(dirname(__DIR__).'/.env');
```

Could you test that it works and if so send a pull request to update the documentation? That would be super appreciated :)

**Comment by Nyholm** [comment]
This is fixed in #37
